Orange/widgets/data/oweditdomain.py

```diff
@@ -1624,9 +1630,16 @@ def initStyleOption(self, option, index):
             option.font.setItalic(True)
 
         multiplicity = index.data(MultiplicityRole)
+        warnings_ = index.data(RestoreWarningRole)
+
+        def set_color(palette: QPalette, color):
+            palette.setBrush(QPalette.Text, QBrush(color))
+            palette.setBrush(QPalette.HighlightedText, QBrush(color))
+
         if isinstance(multiplicity, int) and multiplicity > 1:
-            option.palette.setBrush(QPalette.Text, QBrush(Qt.red))
-            option.palette.setBrush(QPalette.HighlightedText, QBrush(Qt.red))
+            set_color(option.palette, Qt.red)
+        elif warnings_:
+            set_color(option.palette, Qt.yellow)
```

Comment:
Just a minor comment: yellow looks nice in dark mode, but is difficult to see on white background. Could you change it to a color that is visible in both modes, or choose different colors for different modes?

<img width="295" alt="Screenshot 2024-09-20 at 13 57 03" src="https://github.com/user-attachments/assets/ab7895a2-2c24-47af-8373-486ec484694b">